### ISSUE TITLE
Syntax clarification 

### ISSUE THREAD

**Issue by kornai** [kornai/4lang#161]
The definition syntax uses "< >" enclosed parts in about 10% of the cases. I think this refers to default but negatable values, as in title	ci1m	appellatio	tytul1	371		N	name, <book> HAS or wine	bor	vinum	wino	332		N	drink, before(fruit), before(<grape>), alcohol IN/2758

Is the logic of when to apply <> described somewhere?

**Comment by makrai** [comment]
Yes, angled brackets signify default status, and they don't need to be compatible:

|||||||||
|-|-|-|-|-|-|-|-|
|can|konzerv|cibus_conservatus|puszka|1427|u|N|cylinder, metal, CONTAIN [<food>,<drink>]|

**Comment by makrai** [comment]
(sorry for closing. I think in the case of a question issue, it is the questioner supposed to close it)
